src/kudu/consensus/quorum_util.h

```diff
@@ -49,6 +49,8 @@ bool IsRaftConfigMember(const std::string& uuid, const RaftConfigPB& config);
 bool IsRaftConfigVoter(const std::string& uuid, const RaftConfigPB& config);
 bool GetRaftConfigMemberRegion(const std::string& uuid, const RaftConfigPB& config,
     bool *is_voter, std::string *region);
```

Comment:
This function is still needed. It is called by MysqlRaft.cpp, and used for master ban, which is still based on region instead of quorum_id